src/backend/functions/work-packages-edit.ts

```diff
@@ -0,0 +1,369 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import middy from '@middy/core';
+import jsonBodyParser from '@middy/http-json-body-parser';
+import httpErrorHandler from '@middy/http-error-handler';
+import validator from '@middy/validator';
+import { descBulletConverter } from './projects';
```

Comment:
I don't believe this will actually work, but I can test it out to find out if you'd like. One of the tough parts about using a serverless backend is the need for code isolation. The way serverless works is essentially that you take an isolated bundle of code and make a random computer run it. If relative path imports were allowed / worked, then that isolated bundle would have to include all the files that the main file references. This would quickly break down. This is why we built the `utils` package to share code between the backend lambda endpoints because the serverless bundling will pull in package imports. Thus, everything in the `utils` package is included in every backend code bundle. That said, we can't put code that relies on Prisma into the `utils` package because the `utils` package has its own package.json. TL;DR you have to copy paste and duplicate the code, you can't export / import it

Reply:
oh huh that's interesting, ok I'll just bring it to this file then